### ISSUE TITLE
Show precipitation amount as bar chart and ONLY probability as number (rather than probability + precipitation amount as two numbers).

### ISSUE THREAD

**Issue by ncd7** [mlamberts78/weather-chart-card#178]
**Is your feature request related to a problem? Please describe.**
No, this is a pure feature request.

**Describe the solution you'd like**
This is for the forecast section.
I'd love to be able to only show rainfall probability as a number and the bar charts to reflect the expected precipitation amount. I don't want to see the rainfall amount as a number as well.
Right now, the closest option is to select precipitation type rainfall, and then to enable show probability but I don't need to see the precipitation number in that case, it's redundant since the bar chart will already shows the rainfall amount. 

**Additional context**
**This will help show cleanly the two independent values related to rainfall: probability as a number and rainfall amount as the bar chart, without any duplication of data**

Thank you.

**Comment by jbarneson** [comment]
i agree, but would offer more flexability between rainfall amount and probability.. offer charting either one, or displaying the number for either / or. 

Further enhancement, I'd love to get an hourly graph, but daily number

ex: i'm graphing probability hourly, but i don't need to see hourly amounts.. a daily total would suffice: 

<img width="533" alt="image" src="https://github.com/user-attachments/assets/0870331c-649b-49b3-a1b7-f760be217f02">
